### DISCUSSION TITLE
1120 ver can't play any multi room

### DISCUSSION THREAD

**Discussion by Loreos7** [ppy/osu#15691] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/86934170/142718704-7c88dec6-06c6-4368-9474-ae71b123112c.png)

https://user-images.githubusercontent.com/86934170/142718731-cdaa8e5b-dff9-4ae5-b6cb-4b3820e770d5.mp4

[database.log](https://github.com/ppy/osu/files/7574154/database.log)
[network.log](https://github.com/ppy/osu/files/7574155/network.log)
[performance.log](https://github.com/ppy/osu/files/7574156/performance.log)
[runtime.log](https://github.com/ppy/osu/files/7574157/runtime.log)
[updater.log](https://github.com/ppy/osu/files/7574158/updater.log)

**Comment by nahkd123** [comment]
The update only released few hours ago, please wait for a while then try again (the server is likely not updated to accept your client)

**Comment by peppy** [comment, marked as answer]
Please try again now. Had some issues with deployment

**Reply by Loreos7** [reply]
Yep. It's working now.
